app/assets/javascripts/users.coffee

```diff
@@ -0,0 +1,9 @@
+ready = ->
```

Comment:
I gotta be honest, being able to lock the entire set of users all at once kinda skeezes me out - worried about a bad actor worst case scenario, and I can't really think of a time when someone would need to mass reset passwords like that. Can we comment it out for now and create an issue on how to deal with it?